sites/fast-website/src/public/index.html

```diff
@@ -3,11 +3,14 @@
   <head>
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
     <title><%= htmlWebpackPlugin.options.title %></title>
   </head>
   <body>
-    <fast-design-system-provider use-defaults>
+    <fast-design-system-provider
```

Comment:
should this be the `website-design-system-provider`?

Reply:
I wasn't sure if we should use separate buckets or buckets of the same type. Does it make any real difference when they're nested?

Reply:
I guess it depends on the requirements - what requirement is driving toward using two different provider types?